pdns/recursordist/rec-main.cc

```diff
@@ -2216,194 +2218,201 @@ class PeriodicTask
   const string name;
 };
 
-static void houseKeeping(void*)
+static void houseKeeping0(Logr::log_t log)
```

Comment:
what does 0 mean here?

Reply:
`houseKeeping0` is the body of the original main loop of `houseKeeping`.  I agree it could be named better.